test/DynamoCoreTests/TestZeroTouchClass.cs

```diff
@@ -72,5 +74,107 @@ public void DescriptionTest()
             node = new DSFunction(fucDescriptor);
             Assert.AreEqual(fucDescriptor.Signature, node.Description);
         }
+
```

Comment:
These tests should not be in `TestZeroTouchClass`. These attributes should not be applied to zero touch classes. They tests should be among other `NodeModel` tests.